arviz/plots/autocorrplot.py

```diff
@@ -30,51 +30,56 @@ def plot_autocorr(
     Parameters
     ----------
     data: obj
-        Any object that can be converted to an az.InferenceData object
-        Refer to documentation of az.convert_to_dataset for details
+        Any object that can be converted to an :class:`arviz.InferenceData` object
+        refer to documentation of :func:`arviz.convert_to_dataset` for details
     var_names: list of variable names, optional
-        Variables to be plotted, if None all variable are plotted. Prefix the
-        variables by `~` when you want to exclude them from the plot. Vector-value
+        Variables to be plotted, if None all variables are plotted. Prefix the
+        variables by ``~`` when you want to exclude them from the plot. Vector-value
         stochastics are handled automatically.
     filter_vars: {None, "like", "regex"}, optional, default=None
         If `None` (default), interpret var_names as the real variables names. If "like",
         interpret var_names as substrings of the real variables names. If "regex",
         interpret var_names as regular expressions on the real variables names. A la
-        `pandas.filter`.
+        ``pandas.filter``.
     max_lag: int, optional
-        Maximum lag to calculate autocorrelation. Defaults to 100 or num draws, whichever is smaller
-    combined: bool
-        Flag for combining multiple chains into a single chain. If False (default), chains will be
+        Maximum lag to calculate autocorrelation. Defaults to 100 or num draws, whichever is smaller.
```

Comment:
This line is too long, you have to make it be 100 or less characters, then we can merge as all checks will pass

Reply:
it is 93 characters.